src/dependency-manager/src/manager.ts

```diff
@@ -359,6 +375,7 @@ export default abstract class DependencyManager {
         context[normalized_ref].architect = architect_context;
       }
 
+      // TODO:84:add support for references to tasks
```

Comment:
Add another todo to the register cmd to build/push images if there is a build context.